From vapier@gentoo.org  Thu Apr 22 16:26:35 2010
From: Mike Frysinger <vapier@gentoo.org>
Date: Tue,  9 Mar 2010 12:25:40 -0500
Subject: serial: bfin_sport_uart: drop redundant cpu depends
To: linux-serial@vger.kernel.org, Alan Cox <alan@lxorguk.ukuu.org.uk>
Cc: Andrew Morton <akpm@linux-foundation.org>, linux-kernel@vger.kernel.org, uclinux-dist-devel@blackfin.uclinux.org
Message-ID: <1268155540-8241-14-git-send-email-vapier@gentoo.org>


The BF54xM procs imply the related BF54x define, so no need to check both.

Signed-off-by: Mike Frysinger <vapier@gentoo.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/serial/Kconfig |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/serial/Kconfig
+++ b/drivers/serial/Kconfig
@@ -1439,7 +1439,7 @@ config SERIAL_BFIN_SPORT_CONSOLE
 
 config SERIAL_BFIN_SPORT0_UART
 	bool "Enable UART over SPORT0"
-	depends on SERIAL_BFIN_SPORT && !(BF542 || BF542M || BF544 || BF544M)
+	depends on SERIAL_BFIN_SPORT && !(BF542 || BF544)
 	help
 	  Enable UART over SPORT0
 
